datacommons/places.py

```diff
@@ -92,3 +92,59 @@ def get_places_in(dcids, place_type):
   if isinstance(dcids, pd.Series):
     return pd.Series([result[dcid] for dcid in dcids])
   return result
+
+
+def get_pop_obs(dcid):
+  """ Returns :obj:`Place`'s population and observations.
+
+  Args:
+    dcids (:obj:`str`): Dcid of the place.
+
+  Returns:
+    An dictionary of population and observation related to this place.
```

Comment:
"A :obj:\`dict\` of :obj:\`StatisticalPopulation\`'s and :obj:\`Observation\`'s that are associated to the place identified by the given :code:\`dcid\`. The given dcid is the :obj:\`location\` of the returned :obj:\`StatisticalPopulation\`'s, which are the :obj:\`observedNode\`'s of the returned :obj:\`Observation\`'s. See example below for more detail about how the returned :obj:\`dict\` is structured."

We may want to link to the browser nodes for StatisticalPopulation, Observation and location.